webapp/components/create_comment.jsx

```diff
@@ -586,6 +587,7 @@ export default class CreateComment extends React.Component {
                     <span
                         className={'fa fa-smile-o icon--emoji-picker emoji-rhs'}
                         onClick={this.toggleEmojiPicker}
+                        onMouseOver={() => EmojiPicker.beginPreloading()}
```

Comment:
This can just be `onMouseOver={EmojiPicker.beginPreloading}` to prevent creating a function on every render